pandas/tests/io/formats/test_to_csv.py

```diff
@@ -563,3 +563,23 @@ def test_to_csv_compression(self, compression_only,
             result = pd.read_csv(path, index_col=0,
                                  compression=read_compression)
             tm.assert_frame_equal(result, df)
+
+
+
+def test_csv_formatter_line_terminator_default(monkeypatch):
+    # see GH #23608
+    # ensure default line_terminator used is os.linesep
+    df = tm.makeDataFrame()
```

Comment:
I suppose it doesn't largely matter but do you need to call `makeDataFrame` here or could you just construct an empty data frame? The former signals an expectation that we would be doing something with the values which isn't the case in this test

Reply:
I'm pretty sure an empty dataframe would work. I can make that change tonight.